Modify S2083(multiple languages): Update to the education framework (APPSEC-188) (#1328)
SonarSource/rspec: rules/S2083/common/fix/code-rationale.adoc

> The following code is vulnerable to path injection as it is constructing a path
> using untrusted data.
> 
> This path is then used to delete a file without being
> validated first. +
> Therefore, it can be leveraged by an attacker to {code_impact} arbitrary files.
> 

Author: ```suggestion
The following code is vulnerable to path injection as it creates a path using
untrusted data without validation.

An attacker can exploit the vulnerability in this code to {code_impact}
arbitrary files.
```
